web/crux/src/app/notification/notification.mapper.ts

```diff
@@ -1,98 +1,58 @@
-import { Injectable } from '@nestjs/common'
+import { BadRequestException, Injectable } from '@nestjs/common'
 import { Identity } from '@ory/kratos-client'
-import { Notification, NotificationEvent, NotificationEventTypeEnum, NotificationTypeEnum } from '@prisma/client'
-import { InternalException, InvalidArgumentException } from 'src/exception/errors'
-import {
-  AuditResponse,
-  NotificationDetailsResponse,
-  NotificationEventType,
-  NotificationResponse,
-  NotificationType,
-  notificationTypeFromJSON,
-  notificationTypeToJSON,
-} from 'src/grpc/protobuf/proto/crux'
+import { Notification, NotificationEvent, NotificationEventTypeEnum } from '@prisma/client'
 import { nameOrEmailOfIdentity } from 'src/shared/models'
+import { NotificationDetailsDto, NotificationDto, NotificationEventTypeDto } from './notification.dto'
 
 @Injectable()
 export default class NotificationMapper {
-  listItemToProto(notification: NotificationWithEvents): NotificationResponse {
+  listToDto(notification: Notification, createdByIdentity: Identity): NotificationDto {
     return {
-      ...notification,
-      type: this.typeToProto(notification.type),
-      audit: AuditResponse.fromJSON(notification),
-      events: notification.events.map(ev => this.eventTypeToProto(ev.event)),
+      id: notification.id,
+      name: notification.name,
+      url: notification.url,
+      type: notification.type,
+      active: notification.active,
+      createdBy: nameOrEmailOfIdentity(createdByIdentity),
```

Comment:
`createdBy is used everywhere as an id, this is confusing IMHO. Use `creator` or `creatorName` or something.